src/components/common/buttons/CustomIcon.js

```diff
@@ -4,21 +4,18 @@ import { View } from 'react-native'
 import Icon from '../view/Icon'
 import { withStyles } from '../../../lib/styles'
 
-const CustomIcon = ({ styles, theme, name, color, size, borderRadius = 8, wrapper = true, reverse, reverseColor }) => {
-  const wrapperSize = useMemo(() => (wrapper ? borderRadius * 2 : null), [wrapper, borderRadius])
-
+const CustomIcon = ({ styles, theme, name, color, size, circleSize = 16, wrapper = true, reverse, reverseColor }) => {
```

Comment:
We can't use borderRadius from props as we don't pass it, using circleSize instead

Reply:
wrapper prop isn't used. we shouldn 't set width/height if wrapper = false

Reply:
should be 

```javascript
 wrapper && {
   width: circleSize,
   height: circleSize,
 }
```